src/legacy/core_plugins/input_control_vis/public/components/editor/options_tab.test.js

```diff
@@ -18,78 +18,58 @@
  */
 
 import React from 'react';
-import sinon from 'sinon';
 import { shallow } from 'enzyme';
 import { mountWithIntl } from 'test_utils/enzyme_helpers';
 
 import {
   OptionsTab,
 } from './options_tab';
 
-const scopeMock = {
-  editorState: {
-    params: {
-      updateFiltersOnChange: false,
-      useTimeFilter: false
-    }
-  }
-};
-let stageEditorParams;
+describe('OptionsTab', () => {
+  let props;
 
-beforeEach(() => {
-  stageEditorParams = sinon.spy();
-});
+  beforeEach(() => {
+    props = {
+      vis: {},
+      stateParams: {
+        updateFiltersOnChange: false,
+        useTimeFilter: false
+      },
+      setValue: jest.fn()
+    };
+  });
 
-test('renders OptionsTab', () => {
-  const component = shallow(<OptionsTab
-    scope={scopeMock}
-    editorState={scopeMock.editorState}
-    stageEditorParams={stageEditorParams}
-  />);
-  expect(component).toMatchSnapshot(); // eslint-disable-line
-});
+  it('should renders OptionsTab', () => {
+    const component = shallow(<OptionsTab {...props} />);
 
-test('updateFiltersOnChange', () => {
-  const component = mountWithIntl(<OptionsTab
-    scope={scopeMock}
-    editorState={scopeMock.editorState}
-    stageEditorParams={stageEditorParams}
-  />);
-  const checkbox = component.find('[data-test-subj="inputControlEditorUpdateFiltersOnChangeCheckbox"] input[type="checkbox"]');
-  checkbox.simulate('change', { target: { checked: true } });
-  const expectedParams = {
-    updateFiltersOnChange: true
-  };
-  sinon.assert.calledOnce(stageEditorParams);
-  sinon.assert.calledWith(stageEditorParams, sinon.match(expectedParams));
-});
+    expect(component).toMatchSnapshot(); // eslint-disable-line
```

Comment:
Why is this line disabled for eslint?

Reply:
Seems something legacy and could be removed